### ISSUE TITLE
SAP NetWeaver Monitoring Issue

### ISSUE THREAD

**Issue by RajaKavila** [Azure/Azure-Monitor-for-SAP-solutions-preview#48]
Hi Team,
We have successfully added resource providers for 3 SAP NetWeaver systems under Providers i.e., S01 (Sandbox), D01 (Dev) and R01 (Testing). However, in the Monitoring section of SAP NetWeaver, when we go to the different sub sections, we see that we can see the data only for 1 SAP system (S01) and for the remaining systems (D01 and R01) no data is displayed for some if the sections. 

Data is displayed for all systems (S01, D01 and R01) for the following parameters ... Availability, Enqueue Statistics and Queue Statistics.
Data is displayed only for S01 for the following parameters ... Application Performance, System Performance and Workload Statistics.
Subscription ID: a6124a65-6960-437b-9cc9-cc0ab172df46

We have done the same configuration settings for all the systems. We have rebooted the services, system, removed the systems from Providers and re-added it but no change. We have got the configuration reviewed by MS SME over screen sharing and no config issue is observed. 

We have a go-live next week, and this issue is a showstopper. Please help me fix this issue.
Let me know if you need any other details from my end.

Regards,
Raja Sekhar

**Comment by devanshjainms** [comment]
Hi @RajaKavila ,

Thank you for reaching out to us. I could check from the logs that there may have a few app server(s) that are offline in your landscape. This is a known issue for us where if you have any application server offline, we are unable to collect RFC metrics from the source system, the fix would be in production with in next few weeks.

To unblock you, the work around is that you create a new NetWeaver provider with the "Application Server" property using hostname/fully qualified domain name, and not the IP address.

![image](https://user-images.githubusercontent.com/86314060/194618218-8ec2d50d-f36a-4aef-a53e-f0526e2e4ce9.png)


Rest assured I am happy to get on a call with you if you still have questions.

**Comment by RajaKavila** [comment]
Hi Devansh,
We have tried adding using FQDN but we are getting following error when trying to add using hostname. Please check and assist with the next steps.

<img width="345" alt="image" src="https://user-images.githubusercontent.com/115222104/194806854-08eb725e-67c2-44f1-9364-8619fe9a5ad1.png">

Let me know if you need any other details from our end.

**Comment by RajaKavila** [comment]
Hi Devansh,
Can we get into a call to check this?

**Comment by RajaKavila** [comment]
Hi Devansh, 
We have a go live this weekend starting from Friday and this is a pre-requisite for the go-live to happen. Go-live will be in jeopardy if this is not resolved before that. Hence, this is kind of super critical for us. I have already reached out to MS SMEs, and we are having daily calls, but they are also pointing to get help from you. 

Hence, I need to have a call with you at the earliest feasible today itself. We work in IST time zone. If you can share your earliest time, you are available in IST, we can have a call.

**Comment by RajaKavila** [comment]
Hi @siagrawal,
I just received your message. It is already 10mins past 2:30pm. We can have the call now. Will you be sending the invite to me?

**Comment by siagrawal** [comment]
@RajaKavila 
Scheduled a meeting for 3:30 PM IST,
Microsoft Teams meeting 
Join on your computer, mobile app or room device 
[[Click here to join the meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_MDM0NTI5YjUtZDE0ZS00MTUxLTg5MjEtMzk2YjQ3MGRiZjNm%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22faca1984-b2e3-44ff-b260-90c2a5bdee01%22%7d)] 
Meeting ID: 248 172 865 389 
Passcode: o2QBFK 
[Download Teams](https://www.microsoft.com/en-us/microsoft-teams/download-app) | [Join on the web](https://www.microsoft.com/microsoft-teams/join-a-meeting)

**Comment by siagrawal** [comment]
@RajaKavila 
3:00 PM works, please join

**Comment by edwardmeade1** [comment]
As discussed in call I am closing this ticket. Issue being discussed in other forum

Kind Regards

**Comment by RajaKavila** [comment]
Hi @edwardmeade1 
Do not close this ticket. We never agreed to close the ticket. Devansh was working with me on the same ticket.

**Comment by RajaKavila** [comment]
Hi @edwardmeade1 
Reopen this ticket and confirm. It is not ok to close a ticket without solving an issue. There is no other forum this issue is being discussed upon.

**Comment by edwardmeade1** [comment]
Hi @RajaKavila ,

We had a call today with rest of your team at 8pm IST. There we saw that as FQDN details were entered correctly, that SID S01, D01 and R01 are currently working as expected. Only current issue is for SID D02 with is an authentication issue. There is an action on your team to validate username and password details.

Your team have raised a ticket with our support partners. The rest of your team confirmed it was okay for us to resolve this issue, as it is a duplicate of issue raised by them with our support partners.

Sorry if any confusion has made however we are actively working with your team.

Kind Regards

**Comment by RajaKavila** [comment]
Hi @edwardmeade1,
Thanks for the update. I am sorry that I was not aware of the call at 8pm IST as I was in another call, and I was still referring to our call today morning. My bad and thanks for the help so far. For the 3 systems, S01, D01 and R01, things look good so far.

You can keep this ticket closed. I will raise a new ticket should I need any other help.
